clients/dos/src/main/kotlin/DosClient.kt

```diff
@@ -120,21 +120,24 @@ class DosClient(private val service: DosService) {
     }
 
     /**
-     * Get scan results for a list of [purls]. In case multiple purls are provided, it is assumed that they all refer to
-     * the same provenance (like a monorepo). If [fetchConcluded] is true, return concluded licenses instead of detected
-     * licenses. Return either existing results, a "pending" message if the package is currently being scanned, a
-     * "no-results" message if a scan yielded no results, or null on error.
+     * Get scan results for a list of [packages]. In case multiple packages are provided, it is assumed that they all

```

Comment:
Commit message nits:
- "new Package rows" -> "new package rows"
- "in *the* DOS database"